packages/postcss-configs/tests/defineConfig.test.ts

```diff
@@ -0,0 +1,46 @@
+import type { Config } from "postcss-load-config";
+import { defineConfig, type PostcssConfigTransformer } from "../src/index.ts";
+
+// NOTE: Since the provided configs are transformed into a PostcssEnv plugin instance, we cannot test the resulting config options.
+// Therefore, only the transformers are tested.
+
+test("when a transformer is provided, the transformer is applied on the tsup config", () => {
+    const weirdPluginsTransformer: PostcssConfigTransformer = (config: Config) => {
```

Comment:
No issue here, but FYI `: Config` is not required since you defined PostcssConfigTransformer

Reply:
Fixed thanks